sktime/markov/_base.py

```diff
@@ -266,19 +270,22 @@ def score_cv(estimator: _MSMBaseEstimator, dtrajs, lagtime, n=10, count_mode="sl
         dynamics simulation. J. Chem. Theory Comput. 11, 5002-5011 (2015).
 
     """
-    from sktime.markov import TransitionCountEstimator
     from sktime.util import ensure_dtraj_list
     dtrajs = ensure_dtraj_list(dtrajs)  # ensure format
     if count_mode not in ('sliding', 'sample'):
         raise ValueError('score_cv currently only supports count modes "sliding" and "sample"')
     sliding = count_mode == 'sliding'
     scores = []
     for fold in range(n):
-        dtrajs_split = blocksplit_dtrajs(dtrajs, lag=lagtime, sliding=sliding, random_state=random_state)
+        if blocksplit:
+            dtrajs_split = blocksplit_dtrajs(dtrajs, lag=lagtime, sliding=sliding, random_state=random_state)
+        else:
+            dtrajs_split = dtrajs
+            if len(dtrajs_split) <= 1:
+                raise ValueError("Need at least two trajectories if blocksplit is not used to decompose the data.")
```

Comment:
why is this the case? and should this be mentioned in the `Parameters`/`blocksplit` part of the method docs then?

Reply:
should probably be mentioned. if you dont blocksplit then individual dtrajs are used for train/ validation sets. If you only have one dtraj, there can be no validation

Reply:
oh, that makes total sense (obviously), just a few words would be really helpful! like `Need at least two trajectories to perform cross-validation` ...

Reply:
Incorporated into the docstring.